README.md

```diff
@@ -1,5 +1,9 @@
 # Operating Systems (OER)
 
+[![](https://img.shields.io/static/v1?label=license&message=CCBY-SA4.0&color=%23385177)](https://github.com/open-education-hub/operating-systems/blob/master/COPYING.md)
+[![](https://img.shields.io/badge/users-93-7589D0?logo=discord)](www.bit.ly/OpenEduHub)
+[![](https://img.shields.io/github/contributors/open-education-hub/operating-systems)](https://github.com/open-education-hub/operating-systems/graphs/contributors)
+
```

Comment:
Maybe also add links to our social media accounts [1]?
[1] https://shields.io/category/social

Reply:
Done. I also added Youtube, Twitter and Reddit.

For Youtube I need Channel ID, I left it as a TODO.

Ping @lizababu  for it.

Reply:
Here's the full list of our social media websites. Add as many of them as you want / can:
* Facebook: https://www.facebook.com/openeducationhub
* Linkedin: https://www.linkedin.com/company/open-education-hub
* Twitter: https://twitter.com/OpenEduHub
* Reddit: https://www.reddit.com/user/Open_Education_Hub
* Youtube: https://www.youtube.com/channel/UCumS6d-kaVXreY46eZLtEvA
* Discord: https://bit.ly/OpenEduHub (https://discord.gg/UXjumUSGHx)
* Google Group: https://groups.google.com/g/open-education-hub/about

Reply:
The YouTube channel can be accessed using this link: https://youtube.com/@OpenEduHub

The channel ID is `OpenEduHub`.

Reply:
@teodutu 

For the other platforms, there is no support in shields.io.
I would keep only what we have right now and add it more later mayve.